numbers.js

```diff
@@ -10,14 +10,12 @@ function newRandomNumber(){
 
 var generateRandomNumber = newRandomNumber();
 
-
-
 function checkRandomNumber() {
   var guessFeedback = document.querySelector(".message");
   var numberGuess = recentGuess.value;
   if (numberGuess > 100) {
     alert("Number 1-100 please");
-    display.innerText = "";
+    // display.innerText = "";
```

Comment:
I don't believe you need this line of code. Try it without and reassess if you are really using it.